packages/flutter/lib/src/rendering/sliver_group.dart

```diff
@@ -168,3 +168,147 @@ bool _assertOutOfExtent(double extent) {
   }
   return true;
 }
+
+/// A sliver that places multiple sliver children in a linear array along the
+/// main axis.
+///
+/// Typically, the slivers will be laid out one at a time. Slivers that have been
```

Comment:
In what case would this order of layout not occur?

Reply:
I've updated the documentation to remove this line.